bin/project_local_granule.py

```diff
@@ -125,8 +125,8 @@ def project_granule(
                         {
```

Comment:
A more general question for @owenlittlejohns, but why is this file in `bin` and not the `swath_projector` directory?

Reply:
I think it's just a way to run a single granule without the entire service around it.  So it's more of a developer tool.

Reply:
√

Reply:
Yup - I can't remember the full heritage of this Python module, but I think having it in the `bin` gives it a bit of a clearer status as a developer item, and keeps the main root of the repository nice and clean.